File: Components/ASP.NET/VB/HotelBooking.Tablet/App_Code/SpecialOffer.vb
Imports HotelBooking
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Public Class SpecialOffer
    Public Sub New(ByVal city As City, ByVal hotel As Hotel)
        Me.City = city
        Me.Hotel = hotel
    End Sub
    Public Property City() As City
    Public Property Hotel() As Hotel
End Class

File: Components/ASP.NET/CS/HotelBooking.Tablet/App_Code/SpecialOffer.cs
using HotelBooking;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class SpecialOffer {
    public SpecialOffer(City city, Hotel hotel) {
        City = city;
        Hotel = hotel;
    }
    public City City { get; set; }
    public Hotel Hotel { get; set; }
}
